modality-probe-cli/src/log/mod.rs

```diff
@@ -222,6 +234,7 @@ fn print_as_graph<W: WriteIo>(
     mut stream: W,
 ) -> Result<(), Box<dyn std::error::Error>> {
     // Edges waiting to be drawn.
+    // TODO: use increment / prev for these before moving on to testing.
```

Comment:
Is this something we need a ticket for?

Reply:
It's actually done! I just forgot to remove the TODO.